$NetBSD: patch-build-aux_config.rpath,v 1.1 2020/08/17 07:59:55 tnn Exp $

This leaks into the pkgconfig file and the space separator
confuses pkgsrc wrappers when building dependent packages.

--- build-aux/config.rpath.orig	2017-03-07 13:51:20.000000000 +0000
+++ build-aux/config.rpath
@@ -173,7 +173,7 @@ if test "$with_gnu_ld" = yes; then
   # here allows them to be overridden if necessary.
   # Unlike libtool, we use -rpath here, not --rpath, since the documented
   # option of GNU ld is called -rpath, not --rpath.
-  hardcode_libdir_flag_spec='${wl}-rpath ${wl}$libdir'
+  hardcode_libdir_flag_spec='${wl}-rpath,$libdir'
   case "$host_os" in
     aix[3-9]*)
       # On AIX/PPC, the GNU linker is very broken
